Basic/Basic/RequestManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  RequestManager.swift
+//  Basic
+//
+//  Created by Dima Bondar on 17.08.17.
+//  Copyright © 2017 Dima. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+class RequestManager
+{
+    static func fetchOrganizations(_ getResult: @escaping (_ firt: [String : Any]?)->())
```

Comment:
Code style, should be ` static func fetchOrganizations(_ getResult: @escaping (_ firt: [String : Any]?) -> ())`